src/python/tensorflow_cloud/core/tests/unit/containerize_test.py

```diff
@@ -30,14 +30,17 @@
     VERSION = "latest"
 
 
+@mock.patch("tensorflow_cloud.core.containerize.requests")
 class TestContainerize(unittest.TestCase):
-    def setup(self):
+    def setup(self, mock_requests):
         self.entry_point = "sample.py"
         self.chief_config = machine_config.COMMON_MACHINE_CONFIGS["K80_1X"]
         self.worker_config = machine_config.COMMON_MACHINE_CONFIGS["K80_1X"]
         self.entry_point_dir = "."
         self.mock_registry = "gcr.io/my-project"
         self.project_id = "my-project"
+        mock_requests.get.return_value = mock.Mock()
```

Comment:
Another way would have been to define mock inside of setUp()

```
def setUp(self):
  self._mock_request_get = mock.patch('request.get')
  self._mock_request_get.return_value =  ...   # assign mocked return values.
  self._mock_request_get.start() 

def tearDown(self):
  mock.patch.stopall()
  super(.., ).tearDown()

```

Then, you'd override this mock in test method only necessary.

This way, you didn't need to add mock_requests arg to every single test method.

Reply:
Thank you for the pointer. Updated the test file.